daq/faucet_event_client.py

```diff
@@ -65,12 +69,25 @@ def has_event(self, blocking=False):
             if '\n' in self.buffer:
                 return True
             if blocking or self.has_data():
-                data = self.sock.recv(1024).decode('utf-8')
-                with self._buffer_lock:
-                    self.buffer += data
-            else:
+                data = self.sock.recv(2048).decode('utf-8')
```

Comment:
2048 -> NAMED_CONSTANT

Reply:
It was originally 1024. I changed it to 2048 to test some things but forgot to change back. Do you still think this should be a named constant? This constant isn’t very useful.